tools/help.txt

```diff
@@ -389,6 +389,13 @@ Options:
                 constraints.
 
 
+>>> delete-site
+Remove deployed site from Meteor.
```

Comment:
I don't think this makes sense - you can't really remove an app "from Meteor"; probably it would make more sense to say something like "Delete an app deployed with `meteor deploy`"